src/Store/auth-context.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+
+const AuthContext = React.createContext({
+    token: '',
+    isLoggedIn: false,
+    login: (token) => { },
+    logout: () => [],
```

Comment:
Waarom returnen we hier een empty list?

Reply:
Een goede uitleg moet ik je verschuldigd blijven,,,
In ieder geval is het definiëren van de variabelen, maar omdat deze uiteindelijk point naar een handler is er überhaupt geen return value. Waarom [] ipv {}; geen idee, per ongeluk gedaan denk ik en aangezien er geen errors zijn zo laten staan